### ISSUE TITLE
[IGN][]Datahub][Redesign ]Suggestion on Links

### ISSUE THREAD

**Issue by IGNF-Xavier** [geonetwork/geonetwork-ui#771]
In the dataset page
![image](https://github.com/geonetwork/geonetwork-ui/assets/98382923/86b17a2b-efbf-49e8-beea-409f8d738fbc)

1.  Rename _Liens_ with _Ressources_ in order to list tutorials, documentation, links to PDF, etc.
2. Allow anothers ways to display these ressources : list, cards and authorize the choice for default displaying

No contribution by code by IGN

**Comment by jahow** [comment]
@IGNF-Xavier I'm wondering about the reason behind wanting to change the display of the links; could we maybe switch the layout automatically depending on e.g. the number of links? Or is it a matter of preference?

Also for changing the title to "Ressources", this can be done with a custom translation in the datahub config. I wouldn't be in favor of changing it in the code as 1/ it can break other custom translations and 2/ the term "resources" is also often used to describe the actual data or file described by the record

**Comment by IGNF-Xavier** [comment]
@jahow 
No problem with the custom translation in the datahub config, is it already available for all the titles in the datahub?
The change in the display aims to have a list instead of cards by default because there could many things in this sections : tutorials, PDF, etc.

**Comment by jahow** [comment]
> No problem with the custom translation in the datahub config, is it already available for all the titles in the datahub?

Absolutely, although I have to admit that "finding" the key behind the label isn't straightforward and requires looking at the code.

> The change in the display aims to have a list instead of cards by default because there could many things in this sections : tutorials, PDF, etc.

So this is intended to handle the case where there are too many links to be practical in a carousel? In that case I think a preliminary UX work should be done to find the best way to handle this. I wouldn't be in favor of offering options to change this part of the layout as the default Datahub layout should offer the best possible UX in any known scenario.
